README.md

```diff
@@ -223,6 +259,26 @@ Service Discovery is automatically identifying and locating available services i
 
 Samples are individual data points in a time series. Metrics are collected as samples at regular intervals and are used to analyze the behavior of a system over time. Many time series databases use samples as billing metrics.
 
+### Sampling 
+
+Sampling is the process of collecting a representative subset of data from a larger set. In the context of monitoring systems, sampling can be used to reduce the amount of data that needs to be collected and processed, while still providing meaningful insights.
+
+### Service
+
+A service is a software component or system that provides a specific set of functions to other applications or users. In SRE, services are monitored and managed to ensure their reliability and availability.
+
+### Summary
+
+A summary is a statistical representation of a set of data that shows its distribution and summary statistics, such as mean and variance.
```

Comment:
```suggestion
A summary is a Prometheus metric type that can be used for that can be used to monitor latencies (or other distributions like request sizes). They are similar to Histograms. However, unlike histograms, they give you the exact quantile values instead of estimates. They work best when you need the same latency value, such as P99, and do not want to avoid going through the hassles of setting histogram buckets.
```